.rubocop_todo.yml

```diff
@@ -0,0 +1,59 @@
+# This configuration was generated by
```

Comment:
is this file necessary?

Reply:
Please remove it and add this file to gitignore

Reply:
Ok @estebancordoba, thank you